### PR TITLE
Fixed issues with path during deploy to Object Storage from Windows

### PR DESCRIPTION
**Pull request by nanov94**

### CODE REVIEW

File: src/utils/index.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export * from './logger';
```
Comment (nanov94) [inline src/utils/index.ts:1]: Only `path` new one, other were moved to separate files

File: src/utils/logger.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import {ArgvService} from '../services';
+
+function writeLog(msg: string) {
+    const {quiet} = ArgvService.getConfig();
+
+    if (quiet) {
+        return;
+    }
+
+    console.log(msg);
```
Comment (vovcyan) [inline src/utils/logger.ts:11]: Looks like forgotten

File: src/utils/path.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import {Platforms} from '../constants';
+
+const windowsSlash = '\\';
+const linuxSlash = '/';
```
Comment (vovcyan) [inline src/utils/path.ts:4]: I think isn't a good way to use name of platforms for slashes, cause Mac is not a linux, but also using normal slashes.

What about just `slash` and `backslash`?

File: src/constants.ts [hunk 4]
```diff
@@ -22,6 +22,10 @@ export enum Lang {
     EN = 'en',
 }
 
+export enum Platforms {
+    WINDOWS = 'win32',
```
Comment (vovcyan) [inline src/constants.ts:26]: Let's add mac and linux here for consistency

File: src/utils/index.ts (hunk 1 above)
Comment (vovcyan) [reply src/utils/index.ts:1]: Gotcha

File: src/utils/logger.ts (hunk 2 above)
Comment (nanov94) [reply src/utils/logger.ts:11]: I changed nothing in this code 
https://github.com/yandex-cloud/yfm-docs/blob/master/src/utils.ts#L99-L107

File: src/constants.ts (hunk 4 above)
Comment (nanov94) [reply src/constants.ts:26]: Yep, I agree with your concern, however we don't use them at the moment. 

Ok, I will add it

File: src/utils/logger.ts (hunk 2 above)
Comment (vovcyan) [reply src/utils/logger.ts:11]: Sorry, missed that

**Review by vovcyan — APPROVED** [review]
LGTM

**Pull request merged**